scripts/deployment-testnet/deploy.js

```diff
@@ -17,6 +14,9 @@ const {
   calculateInputMaxTxLevels,
 } = require("../../test/hermez/helpers/helpers");
 
+const pathDeployParameters = path.join(__dirname, "./deploy_parameters.json");
+const pathOutputJson = path.join(__dirname, "./deploy_output.json");
+const deployParameters = require(pathDeployParameters);
 
 const INITIAL_WITHDRAWAL_DELAY = 3600; //seconds
```

Comment:
we could set those parameters through `deploy_parameters.json`
We may want to test those variables or just add different circuits size

Reply:
`const bootCoordinatorURL = "https://boot.coordinator.io";` should be also an input parameter

Reply:
true!